django/db/backends/base/features.py

```diff
@@ -221,6 +221,9 @@ class BaseDatabaseFeatures(object):
     # Defaults to False to allow third-party backends to opt-in.
     can_clone_databases = False
 
+    # Does the backend ignore quoted identifiers case?
```

Comment:
I understand the comment in the context of this PR, but I wonder if something like "Does the backend consider quoted identifiers with different casing to be equal?" might be more clear in the long run?